### DISCUSSION TITLE
Include Georgia country into GitHub Sponsors program

### DISCUSSION THREAD

**Discussion by munrocket** [community/community#34083] [Other Feature Feedback, Questions, & Ideas]
A lot of the people from this country want to participate in GitHub Sponsors program but can't right now.
Please add it, nearest country Turkey is already added.

**Comment by X1nto** [comment]
It'd be lovely if GitHub and Microsoft considered adding small countries like Georgia to the supported GitHub Sponsors countries. Heck, if Turkey and Egypt are available, so should Georgia.

**Comment by michaelknyazev** [comment]
I know many people from Georgia who actively contribute to open source and maintain their own personal projects. This feature would be a great support for them.

**Comment by Goo541173** [comment]
> A lot of the people from this country want to participate in GitHub Sponsors program but can't right now.
> 
> Please add it, nearest country Turkey is already added.
